CDP4WebServices.API/Services/Protocol/IQueryParameters.cs

```diff
@@ -71,6 +75,16 @@ public interface IQueryParameters
         /// </summary>
         object RevisionTo { get; set; }
 
+        /// <summary>
+        /// Gets or sets a collection of <see cref="ClassKind"/> to used during the cherry picking request
+        /// </summary>
+        IEnumerable<ClassKind> ClassKinds { get; set; }
+
+        /// <summary>
+        /// Gets or sets a collection of <see cref="Category"/>s shortname to used during the cherry picking request
+        /// </summary>
+        IEnumerable<string> CategoriesShortName { get; set; }
```

Comment:
i doubt using shortnames wil work, they cannot be guaranteed to be unique accross the chain of RDLs. Please refactor to use Guids, feel free to use shortguids